### PR TITLE
PBC-78: Upgrading and configuring for ACP Bazaarvoice

### PR DESCRIPTION
**Pull request by brunoviana-spryker**

### CODE REVIEW

File: config/Zed/oms/DummyPayment01.xml [hunk 1]
```diff
@@ -117,11 +124,12 @@
...
 
         <events>
-            <event name="authorize" timeout="1 second"/>
+            <event name="authorize" timeout="1 second" />
             <event name="pay" manual="true" timeout="1 hour" timeoutProcessor="OmsTimeout/Initiation" command="DummyPayment/Pay"/>
+            <event name="request product review" onEnter="true" command="Order/RequestProductReviews"/>
             <event name="confirm" onEnter="true" manual="true" command="Oms/SendOrderConfirmation"/>
             <event name="skip timeout" manual="true" timeout="30 minute"/>
-            <event name="cancel" manual="true"/>
+            <event name="cancel" manual="true" />
```
Comment (stereomon) [inline config/Zed/oms/DummyPayment01.xml:132]:
```suggestion
            <event name="cancel" manual="true"/>
```

File: config/Shared/config_default.php [hunk 2]
```diff
@@ -187,6 +189,7 @@
     $sprykerFrontendHost,
     'threedssvc.pay1.de', // trusted Payone domain
     'www.sofort.com', // trusted Payone domain
```
Comment (stereomon) [inline config/Shared/config_default.php:191]: I know it is not yours but sofort.com -> Payone??

File: config/Shared/config_default.php [hunk 3]
```diff
@@ -675,20 +678,26 @@
...
+    OrderStatusChangedTransfer::class => 'orders',
 ];
 
 $config[MessageBrokerConstants::CHANNEL_TO_TRANSPORT_MAP] =
 $config[MessageBrokerAwsConstants::CHANNEL_TO_RECEIVER_TRANSPORT_MAP] = [
     'payment' => MessageBrokerAwsConfig::SQS_TRANSPORT,
     'assets' => MessageBrokerAwsConfig::SQS_TRANSPORT,
     'product' => MessageBrokerAwsConfig::SQS_TRANSPORT,
+    'reviews' => MessageBrokerAwsConfig::SQS_TRANSPORT,
+    'orders' => MessageBrokerAwsConfig::SQS_TRANSPORT,
```
Comment (stereomon) [inline config/Shared/config_default.php:691]: Isn't this only needed for sending?

File: composer.json [hunk 4]
```diff
@@ -82,11 +82,17 @@
         "spryker-feature/tax": "^202212.0",
         "spryker-shop/asset-widget": "^1.0.0",
```
Comment (stereomon) [inline composer.json:83]: Are no tests needed on the project level?

**Review by stereomon — CHANGES_REQUESTED** [review]

File: composer.json (hunk 4 above)
Comment (matweew) [reply composer.json:83]: I did not get which tests you mean by commenting on the composer.json

File: composer.json [hunk 5]
```diff
@@ -166,10 +172,12 @@
         "spryker/oauth-company-user": "^2.2.0",
         "spryker/oauth-customer-connector": "^1.8.0",
         "spryker/oauth-dummy": "^1.1.0",
+        "spryker/oms": "^11.23.1",
```
Comment (matweew) [inline composer.json:175]: you've added it to the composer.json 

- spryker/oms
- spryker/payment
- spryker/product-review
- spryker/product-review-gui

These dependencies are already sub-dependency of spryker-feature/sales, spryker-feature/reviews, spryker-feature/payments.

so please do `composer update package:version` not `composer require` for those packages.

File: config/Zed/oms/DummyPayment01.xml [hunk 6]
```diff
@@ -29,6 +29,7 @@
                 <flag>cancellable</flag>
             </state>
             <state name="exported" reserved="true" display="oms.state.exported"/>
+            <state name="product review requested" reserved="true" display="oms.state.paid"/>
```
Comment (matweew) [inline config/Zed/oms/DummyPayment01.xml:32]: if the transition is `paid` -> `product review requested`, please put it after `<state name="paid" ...` to improve visibility.

File: config/Zed/oms/DummyPayment01.xml (hunk 1 above)
Comment (brunoviana-spryker) [reply config/Zed/oms/DummyPayment01.xml]: fixed

File: config/Shared/config_default.php [hunk 7]
```diff
@@ -675,20 +678,26 @@
...
 ];
 
 $config[CartsRestApiConstants::IS_QUOTE_RELOAD_ENABLED] = true;
 $config[MessageBrokerAwsConstants::CHANNEL_TO_SENDER_TRANSPORT_MAP] = [
     'payment' => 'http',
     'assets' => 'http',
     'product' => 'http',
+    'reviews' => 'http',
```
[the remark this answers is not in the document]
Comment (brunoviana-spryker) [reply config/Shared/config_default.php]: fixed

File: config/Shared/config_default.php (hunk 3 above)
Comment (brunoviana-spryker) [reply config/Shared/config_default.php]: fixed

File: composer.json (hunk 5 above)
Comment (brunoviana-spryker) [reply composer.json]: fixed

File: config/Zed/oms/DummyPayment01.xml (hunk 6 above)
Comment (brunoviana-spryker) [reply config/Zed/oms/DummyPayment01.xml]: fixed

**Review by matweew — APPROVED** [review]

**Review by stereomon — APPROVED** [review]

**Pull request merged**